shared/oae/api/oae.api.util.js

```diff
@@ -1866,7 +1866,5 @@ define(['exports', 'require', 'jquery', 'underscore', 'oae.api.config', 'markdow
      *
      * @return {String}   Event(s) that can fire when the content of an input field changes
      */
-    var getFormChangeEventNames = exports.getFormChangeEventNames = function() {
-        return $('html').hasClass('ie-lt10') ? 'change keyup paste cut' : 'change input';
-    };
+    var getFormChangeEventNames = exports.getFormChangeEventNames = return "change input";
```

Comment:
Thanks for that. Well, I think you managed to achieve what you intended, in the master branch. But your real problem is how to apply those extra commits to the feature branch you're working on, is this correct? Currently it should be similar to this:

![screen shot 2017-04-07 at 14 33 58](https://cloud.githubusercontent.com/assets/19879/24802387/baaa450c-1b9f-11e7-945f-1ac06ba35766.png)

If so, you can now use cherry picking, which would go like:

![screen shot 2017-04-07 at 14 34 41](https://cloud.githubusercontent.com/assets/19879/24802420/dd3a8212-1b9f-11e7-94fe-488642850cfd.png)

...or you can just `git reset` back to your latest commit on `master` and then merge that branch onto the feature branch. It would go something like this:

![screen shot 2017-04-07 at 14 42 45](https://cloud.githubusercontent.com/assets/19879/24802663/97e435c2-1ba0-11e7-8c35-ceaa51e7292e.png)

Hope that helps!